zebrad/src/commands/start.rs

```diff
@@ -328,49 +393,133 @@ impl StartCmd {
                     .best_tip_height()
                     .expect("unexpected empty state: estimate requires a block height");
 
+                // Work out the sync progress towards the estimated tip.
                 let sync_progress = f64::from(current_height.0) / f64::from(estimated_height.0);
-                let sync_percent = format!("{:.3}", sync_progress * 100.0);
+                let sync_percent = format!(
+                    "{:.frac$}",
+                    sync_progress * 100.0,
+                    frac = SYNC_PERCENT_FRAC_DIGITS,
+                );
```

Comment:
Optional: formatting tweak to become slightly prettier.

```suggestion
                let sync_percent = format!(
                    "{:.frac$} %",
                    sync_progress * 100.0,
                    frac = SYNC_PERCENT_FRAC_DIGITS,
                );
```